__factories__/clients/github.js

```diff
@@ -45,3 +45,17 @@ export const githubProfileFactory = new Factory().attrs({
     private_repos: 10000,
   },
 });
+
+export const githubRepositoryFactory = new Factory().attrs({
+  hiddenUIComponents: ['console'],
+  instructions: '',
+  isArchived: false,
+  projectKey: '123456789',
```

Comment:
Can we make this a dynamic attribute that generates a project key? In particular if a test needs to generate two projects from this factory, we would expect them to have different keys.